build/Makefile

```diff
@@ -13,12 +13,14 @@
 
 # Define core directory below which everything resides. This is the
 # parent directory of the 'build' directory
-F_MASTER =
+F_MASTER = /home/anewman/summa
```

Comment:
@anewman89 : This is why travis is complaining - you should remove this commit from your PR